src/playground-file-editor.ts

```diff
@@ -226,6 +226,11 @@ const mimeTypeToTypeEnum = (mimeType?: string) => {
     case 'text/javascript':
     case 'application/javascript':
       return 'js';
+    case 'text/jsx':
+      return 'jsx';
+    // tsx uses the same syntax highlighting as JSX
+    case 'text/typescript-jsx':
```

Comment:
I think we should have a distinct `jsx` vs `tsx` filetypes. The fact that both filetypes are supported by the same codemirror syntax highlighter is an implementation detail, so that can just be represented in `getLanguageMode`.

So I think this case should return `tsx`.